src/components/base/link/Link.js

```diff
@@ -137,8 +139,8 @@ Link.propTypes = {
   border: PropType_Link_border,
   className: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
   href: PropType_Link_href,
-  tabIndex: PropTypes.string,
-  tag: PropTypes.oneOf(["a", "span"]),
+  tabIndex: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  tag: PropTypes.string, // support any tag
```

Comment:
👏

Reply:
hvorfor egentlig både PropTypes.string og PropTypes.number for tabIndex? er det ikke mere logisk, hvis det kun var number?

Reply:
før var kun string, det ville kræve en stor refactor. Tænker begge var ok, da det ender med at være en string i DOM'en alligevel - så ikke så forkert med string. Men enig i at det jo kun er "number" scenarier der virker som tabIndex